src/odemis/util/test/spectrum_test.py

```diff
@@ -49,7 +50,82 @@ def test_wl_list(self):
         wl = spectrum.get_wavelength_per_pixel(da)
         self.assertEqual(len(wl), shape[0])
         self.assertEqual(wl, wl_orig)
-        
+
+
+class TestGetAngle(unittest.TestCase):
+
+    def test_angle_per_pixel_simple(self):
+        """
+        Test get_angle_per_pixel when all metadata is good
+        """
+        shape = (512, 256, 1, 5, 4)
+        dtype = numpy.dtype("uint16")
+        wl_orig = (400e-9 + numpy.arange(shape[0]) * 10e-9).tolist()
```

Comment:
These three list can go to the `setUpClass()` if needed as they are common for the test-cases. Just a suggestion.